### DISCUSSION TITLE
[Feature Request] Support for Go

### DISCUSSION THREAD

**Discussion by o-coder-s** [dineug/erd-editor#268] [Ideas]
First of all, Thank you very much for this great extension, God bless your fingers. 

Can you please add support for Code generation for : 
- Go

**Comment by dineug** [comment]
Added @vuerd/plugin-generate-template.
Available in vuerd-vscode v0.9.3.

**Reply by o-coder-s** [reply]
Am using it, I was able to generate 60% of the API by understanding the structure of the JSON file. Thank you.

**Reply by OscarVanL** [reply]
Is go support still included? When I open the 'Generate Template' button on VSCode these are the only options I get.
<img width="861" alt="image" src="https://user-images.githubusercontent.com/7999692/208294527-18733d73-f2af-45da-b829-5b6b542a61cd.png">

**Comment by dineug** [comment]
#424
